### PR TITLE
Ddt 1531 + 1531

### PR DESCRIPTION
**Pull request by FinnNielsen**

### CODE REVIEW

File: edocumentation-transfer-obj/src/main/java/org/dcsa/edocumentation/transferobjects/EquipmentTO.java [hunk 1]
```diff
@@ -7,11 +7,13 @@
 
 import jakarta.validation.constraints.NotNull;
 import jakarta.validation.constraints.Size;
+import org.dcsa.skernel.infrastructure.validation.ISO6346EquipmentReference;
 
 @AllOrNone({"tareWeight", "weightUnit"})
 public record EquipmentTO(
   @NotNull(message = "EquipmentReference is required.")
   @Size(max = 15)
```
Comment (nt-gt) [inline edocumentation-transfer-obj/src/main/java/org/dcsa/edocumentation/transferobjects/EquipmentTO.java:15]: You might as well change size to be `min=11, max=11` here as `@ISO6346EquipmentReference` has those size restrictions (or remove `@Size` and let `@ISO6346EquipmentReference` deal with it)`.

File: edocumentation-transfer-obj/src/main/java/org/dcsa/edocumentation/transferobjects/RequestedEquipmentTO.java [hunk 2]
```diff
@@ -22,7 +28,7 @@ public record RequestedEquipmentTO(
   @NotNull
   Float units,
 
-  List<@NotBlank @Size(max = 15) String> equipmentReferences,
+  List<@NotBlank @Size(max = 15) @ISO6346EquipmentReference String> equipmentReferences,
```
Comment (nt-gt) [inline edocumentation-transfer-obj/src/main/java/org/dcsa/edocumentation/transferobjects/RequestedEquipmentTO.java:31]: You might as well change size to be `min=11, max=11` here as `@ISO6346EquipmentReference` has those size restrictions (or remove `@Size` and let `@ISO6346EquipmentReference` deal with it)`.

**Review by nt-gt — DISMISSED** [review]
Approved but had two minor nits

File: edocumentation-transfer-obj/src/main/java/org/dcsa/edocumentation/transferobjects/EquipmentTO.java (hunk 1 above)
Comment (FinnNielsen) [reply edocumentation-transfer-obj/src/main/java/org/dcsa/edocumentation/transferobjects/EquipmentTO.java]: Point taken `@Size` removed

**Review by nt-gt — APPROVED** [review]

**Pull request merged**